docs/content/docs/KnownIssues.md

```diff
@@ -11,3 +11,17 @@ type: docs
 Disable **Settings > Experimental > Only sync the active variant**
 
 https://stackoverflow.com/questions/54201216/android-studio-3-3-new-gradle-sync-is-not-supported-due-to-containing-kotlin-mo/
+
+## Instrumentation tests run from Android Studio are failed with `java.lang.NoClassDefFoundError`
+
+The problem occurs only when running a test from Android Studio because of generated Run Configuration has inappropriate type.
+Wherein test run from command line via `./gradlew test` ends up normally.  
+Mostly this reproduced in tests that use Kotlin extensions from Test Fixtures from other Gradle modules.
+
+Workaround:
+1. Use IntelliJ IDEA;
+2. Open **Settings > Build, Execution, Deployment > Build Tools > Gradle **;
+3. In drop-down menu **Run test using** select **Gradle** instead of **IntelliJ IDEA**;
+4. Then go to "Edit Configurations..." (which is is the drop-down menu to the right of Run button);
+5. Remove old Run Configuration with "-" button;
+6. Run test again (make sure new configuration is created).
```

Comment:
For ordered lists we use simpler [format](https://avito-tech.github.io/avito-android/docs/contributing/docs/#%D0%BD%D1%83%D0%BC%D0%B5%D1%80%D0%BE%D0%B2%D0%B0%D0%BD%D0%BD%D1%8B%D0%B9-%D0%B8%D0%BB%D0%B8-%D0%BD%D0%B5%D0%BD%D1%83%D0%BC%D0%B5%D1%80%D0%BE%D0%B2%D0%B0%D0%BD%D0%BD%D1%8B%D0%B9) with `1.`

Reply:
Yeah, I forgot about it. Fixed.